metron-analytics/metron-profiler/README.md

```diff
@@ -6,36 +6,94 @@ This is achieved by summarizing the streaming telemetry data consumed by Metron
+    /localrepo/metron-profiler-0.4.0-201707112313.noarch.rpm
+    ```
 
-1. Stand-up a Metron environment.  For this example, we will use the 'Quick Dev' environment.  Follow the instructions included with [Quick Dev](../../metron-deployment/vagrant/quick-dev-platform) or build your own.
+1. Install the RPM.
+
+    ```
+    [root@node1 ~]# rpm -ivh metron-profiler-*.noarch.rpm
+    Preparing...                ########################################### [100%]
+       1:metron-profiler        ########################################### [100%]
+    ```
+
+    ```
+    [root@node1 ~]# rpm -ql metron-profiler
+    /usr/metron
+    /usr/metron/0.4.1
+    /usr/metron/0.4.1/bin
+    /usr/metron/0.4.1/bin/start_profiler_topology.sh
+    /usr/metron/0.4.1/config
+    /usr/metron/0.4.1/config/profiler.properties
+    /usr/metron/0.4.1/flux
+    /usr/metron/0.4.1/flux/profiler
+    /usr/metron/0.4.1/flux/profiler/remote.yaml
+    /usr/metron/0.4.1/lib
+    /usr/metron/0.4.1/lib/metron-profiler-0.4.0-uber.jar
+    ```
+
+1. Create a table within HBase that will store the profile data. The table name and column family must match the [Profiler's configuration](#configuring-the-profiler).  By default, the table is named `profiler` with a column family `P`.
 
-1. Create a table within HBase that will store the profile data. The table name and column family must match the [Profiler's configuration](#configuring-the-profiler).
     ```
     $ /usr/hdp/current/hbase-client/bin/hbase shell
     hbase(main):001:0> create 'profiler', 'P'
     ```
 
-1. Edit the configuration file located at `$METRON_HOME/config/profiler.properties`.  Change the kafka.zk and kafka.broker values from "node1" to the appropriate host name.  Keep the same port numbers:
+1. Edit the configuration file located at `$METRON_HOME/config/profiler.properties`.  
     ```
     kafka.zk=node1:2181
     kafka.broker=node1:6667
     ```
+    Change `kafka.zk` to refer to Zookeeper in your environment.  
+    Change `kafka.broker` to refer to a Kafka Broker in your environment.
+
+1. Start the Profiler topology.
+    ```
+    $ cd $METRON_HOME
+    $ bin/start_profiler_topology.sh
+    ```
+
+At this point the Profiler is running and consuming telemetry messages.  We have not defined any profiles yet, so it is not doing anything very useful.  The next section walks you through the steps to create your very first "Hello, World!" profile.
+
+## Getting Started
 
-1. Define the profile in a file located at `$METRON_HOME/config/zookeeper/profiler.json`.  The following example JSON will create a profile that simply counts the number of messages per `ip_src_addr`, during each sampling interval.
+This section will describe the steps required to get your first "Hello, World!"" profile running.  This assumes that you have successfully [installed the Profiler](#installation) and have it running.
+
+1. Create the profile definition in a file located at `$METRON_HOME/config/zookeeper/profiler.json`.  
```

Comment:
$METRON_HOME/config/zookeeper/profiler.json needs to be created for the first time , it will not exist by default .